.github/workflows/osx.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         ./contrib/utilities/download_clang_format
     - name: info
       run: |
-        g++-9 -v
+        g++-10 -v
```

Comment:
Why do we print this at all if we don't use `gcc` here?

Reply:
I think we do. for 64bits we use mpicxx (based on gcc). I'd be ok to remove it, but I think it is a good idea to keep in the logs what has been used.